react/features/prejoin/actions.js

```diff
@@ -273,6 +448,45 @@ export function setDeviceStatusWarning(deviceStatusText: string) {
     };
 }
 
+/**
+ * Action used to set the dial out status.
+ *
+ * @param {string} value - The status.
+ * @returns {Object}
+ */
+function setDialOutStatus(value: string) {
+    return {
+        type: SET_DIALOUT_STATUS,
+        value
+    };
+}
+
+/**
+ * Action used to set the dial out country.
+ *
+ * @param {Object} value - The country.
```

Comment:
Object? Can you detail what structure it has?

Reply:
Done.